lib/puppet/parser/functions/trocla_get.rb

```diff
@@ -16,10 +16,14 @@ module Puppet::Parser::Functions
 
 Get the mysql style sha1 hashed password.
 
+    $cert_x509_key = trocla_get('cert_x509','x509', 'render: keyonly')
+
+Get the x509 style private key.
+
 By default puppet will raise a parse error if the password haven't yet been
 stored in trocla. This can be turned off by setting false as a third argument:
 
-    $password_user3 = trocla_get('user2','mysql',false)
+    $password_user3 = trocla_get('user2','mysql',nill,false)
```

Comment:
So this not `nill` rather should be nil, also above

And I'm wondering whether we could keep the signature of the function, as this would potentially mean your PR makes it incompatible with any usage out there. But I think I have an idea.